db/schema_tables.cc

```diff
@@ -1512,6 +1540,14 @@ static std::vector<data_type> read_arg_types(const query::result_set_row& row, c
     return arg_types;
 }
 
+static std::vector<data_value> read_arg_values(const query::result_set_row& row) {
```

Comment:
The function should be better named, it's not clear it is specific to scylla_aggregates.

Reply:
I've  based the name on almost identical function just above which has name `read_arg_types` and returns `vector<data_type>`. It also read column named `argument_types`.

Reply:
One badly named function isn't an excuse for another. Also, at least the existing one has a matching function name and column name. The new one is called arg_values but reads a column named argument_types.